exercises/concept/lasagna/.meta/exemplar.cpp

```diff
@@ -0,0 +1,24 @@
+// ovenTime returns the amount in minutes that the lasagna should stay in the
+// oven.
+int ovenTime() { return 40; }
+
+/* RemainingOvenTime returns the remaining
+minutes based on the actual minutes already in the oven.
+*/
+int remainingOvenTime(int actualMinutesInOven) {
+    return ovenTime() - actualMinutesInOven;
+}
+
+/* preparationTime returns an estimate of the preparation time based on the
+   number of layers and the necessary time per layer.
+*/
+int preparationTime(int numberOfLayers) {
+    int prepTimePerLayer{2};
```

Comment:
If you remove this line you can skip dealing the _variables_ concept in the _basics_ node 🎉

Also discussion of statements would become redundant, and reasoning about functions would become easy as they'd all be pure.

Reply:
The student will never see the exemplar.cpp file. I'm not sure if I should make a new concept to explain and getting into the nitty-gritty of definitions, initializations and everything without having them here in this concept first.

It is not necessary to have a variable definition in the function, but it makes the whole thing a bit more structured so they don't get hit with magic numbers.